### DISCUSSION TITLE
Paste Seownedde hitscan to fedware hitscan

### DISCUSSION THREAD

**Discussion by Ragdoll117** [Fedoraware/Fedoraware#1152] [Ideas]
Do it

**Comment by HutchyBen** [comment]
We don't paste here 🤪

**Comment by N1kolasFlaV** [comment]
greta diea

**Comment by Ragdoll117** [comment]
> We don't paste here 🤪
Wow, as if Fed isn't TOTALLY pasted from the original Seowned 
Give more proofs that Fedware is not a paste and I will close this comment

**Comment by BindUser** [comment]
You must give proof that fed is pasted first
